pycytominer/operations/transform.py

```diff
@@ -65,7 +66,7 @@ def __init__(self, epsilon=1e-6, center=True, method="ZCA", return_numpy=False):
             raise ValueError("PCA-cor and ZCA-cor require center=True")
 
     def fit(self, X, y=None):
-        """Identify the sphering transform given self.X
+        """Identify the sphering transform given self.X.
```

Comment:
Would this be true? It looked like `X` is derived from the method's parameters rather than a class attribute.
```suggestion
        """Identify sphering transform given parameter X.
```